### DISCUSSION TITLE
Not sure about the new playlist priority system

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#7574] [Q&A]
First of all, I would like to thank the maintainers of this repo for their efforts to provide us with a complete, up-to-date and free suite for our webradios.

I recently updated to v0.20.3 Stable on Docker.
The operation of my "Jingles" playlists seems strange.

Applying a higher or lower priority than the music playlist weights does not impact the priority at all. The Jingles playlists are no longer played or without any control over them.

Initially, on the old stable version, the Jingles were played every 2 tracks randomly without repetitions (see attachment), but despite different tests by modifying the priorities and weights of the playlists (which seem to be two different things), I cannot get the jingles to work.

![image](https://github.com/user-attachments/assets/e0ccedb4-8434-4304-8910-67bde400107e)

Is this a known malfunction or a configuration error on my part?

Thank you! 😀

**Comment by BusterNeece** [comment]
Moving to Issue: #7575

**Comment by LordHelmchen666** [comment]
Do Not use playlist priority and it should work as before.
If you want to use Playlist Priority give your Jingles higher priority than your other Playlists.
For example i use

   - Default Playlists 1
-    Scheduled Playlists 2
-    Syndicated Shows 3
-    Jingles, Promos 10
-    News 15

IMPORTANT
If you set Priority on one of your Playlists. You must set priority on All Playlists or it will not work.
